eclipse-collections-code-generator/src/main/resources/api/map/mutablePrimitivePrimitiveMap.stg

```diff
@@ -47,6 +48,8 @@ public interface Mutable<name1><name2>Map extends <name1><name2>Map, Mutable<nam
 
     void putAll(<name1><name2>Map map);
 
+    void updateValues(<name1><name2>To<name2>Function update);
```

Comment:
The function name should be `function` or some other noun, rather than `update`.

Reply:
@motlin ok

Reply:
@motlin I've pushed changes for this, see also my other comments